Filtering.md

```diff
@@ -159,3 +159,20 @@ end
 ```
 
 These declarations can also be overridden from the command line.
+
+## Silencing filter announcements
+
+By default, RSpec will print a message before your specs run indicating what
+filters are configured, for instance, it might print
+"Run options: include {:focus=>true}" if you set `config.filter_run_including
+:focus => true`.
```

Comment:
That makes sense. I just modified my commit to combine the lines here and for the second sentence below as well.